packages/labs/ssr-dom-shim/src/test/types.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.css' {
```

Comment:
Let's link to https://github.com/microsoft/TypeScript/issues/46689 and https://github.com/microsoft/TypeScript/issues/46135 so we can hopefully remove this in the future.